### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gcauchon**
## 📖 Description

Many updates were released in the last weeks/months, here a few `lib` updates; getting the boilerplate ready for Elixir `1.11` that will follow next!

- Erlang `23.1`
- Alpine `1.12`
- Ecto `3.5`
- Phoenix `1.5`
- Absinthe `1.5` 

## 📝 Notes

`erlang-crypto` is not required with the update to Alpine `1.12.0`, so I deployed the app to Heroku to validate `FORCE_SSL` and `DATABASE_SSL` are still fully functional!

https://elixir-boilerplate.herokuapp.com

### CODE REVIEW

**Review by garno — APPROVED** [review]

File: config/config.exs [hunk 1]
```diff
@@ -9,9 +9,8 @@ config :elixir_boilerplate,
 config :phoenix, :json_library, Jason
 
 config :elixir_boilerplate, ElixirBoilerplateWeb.Endpoint,
-  pubsub: [name: ElixirBoilerplate.PubSub, adapter: Phoenix.PubSub.PG2],
-  render_errors: [view: ElixirBoilerplateWeb.Errors.View, accepts: ~w(html json)],
-  instrumenters: [NewRelic.Phoenix.Instrumenter]
```
Comment (charlesdemers) [inline config/config.exs:14]: Why do we remove NewRelic?

Comment (gingman) [inline config/config.exs:14]: Why do we remove this line?

Comment (gcauchon) [reply config/config.exs:14]: @gingman ^this

**Review by gingman — APPROVED** [review]

**Review by Boubalou — APPROVED** [review]
Has not Elixir 1.11 been out officially as well?

**Comment by gcauchon** [comment]
> Has not Elixir 1.11 been out officially as well?

Yes, I wanted to do the upgrade in a separate PR on purpose!

**Pull request merged**